From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Wed, 6 Nov 2019 01:42:24 -0500
Subject: Drop unnecessary tests in keyObj-test.js

See Message-ID: <8736f1nfsd.fsf@fifthhorseman.net> (and followup
messages) on enigmail-users@enigmail.net for more discussion about why
these tests are bogus and irrelevant for debian.
---
 package/tests/keyObj-test.js | 17 +----------------
 1 file changed, 1 insertion(+), 16 deletions(-)

diff --git a/package/tests/keyObj-test.js b/package/tests/keyObj-test.js
index 0042e02..a82caa5 100644
--- a/package/tests/keyObj-test.js
+++ b/package/tests/keyObj-test.js
@@ -37,19 +37,4 @@ test(withTestGpgHome(withEnigmail(function shouldExportMinimalSubkey() {
     Assert.equal(minKey.keyData.substr(0, 50), "mQGNBFub08oBDACmb04i4u8xUV1ADbnbN5l83mpr70OyWVJb5E");
     Assert.equal(minKey.keyData.substr(-50, 50), "p9TFNKjguUrrGrVnmnmy/YoGTJWuGqrZy8kcC3LCjg0k2mV0M=");
   }
-
-  EnigmailGpg.agentVersion = "2.2.1";
-  Assert.ok(!EnigmailGpg.getGpgFeature("export-specific-uid"));
-
-  let minKey = keyObj.getMinimalPubKey("bob@somewhere.invalid");
-  Assert.equal(minKey.exitCode, 0);
-  Assert.equal(minKey.keyData.substr(3, 50), "NBFub08oBDACmb04i4u8xUV1ADbnbN5l83mpr70OyWVJb5ElIc");
-  Assert.ok(minKey.keyData.substr(-50, 50) == "1MU0qOC5SusatWeaebL9igZMla4aqtnLyRwLcsKODSTaZXQw==" ||
-    minKey.keyData.substr(-50, 50) == "p9TFNKjguUrrGrVnmnmy/YoGTJWuGqrZy8kcC3LCjg0k2mV0M=", "min key matches");
-
-  minKey = keyObj.getMinimalPubKey("does@not.exist");
-  Assert.equal(minKey.exitCode, 0);
-
-  Assert.equal(minKey.keyData.substr(0, 50), "xsDNBFub08oBDACmb04i4u8xUV1ADbnbN5l83mpr70OyWVJb5E");
-  Assert.equal(minKey.keyData.substr(-50, 50), "1MU0qOC5SusatWeaebL9igZMla4aqtnLyRwLcsKODSTaZXQw==");
-})));
\ No newline at end of file
+})));
